### PR TITLE
Initial Grading updates; fix courseList error

### PR DESCRIPTION
**Pull request by ShaydeNofziger**
Still more grade work to be done but I noticed some issues with getCourseList not returning correctly. This fixes that.

Also sets the response header to JSON as opposed to HTML, so we can actually parse the response as JSON.

### CODE REVIEW

File: API/API.php [hunk 1]
```diff
@@ -983,15 +983,16 @@ function getCourseList()
 		$query = $sqlite->prepare("SELECT * FROM Course");		
 		$result = $query->execute();
 		
-		$record = null;
+		$record = array();
 		//$sqliteResult = $sqlite->query($queryString);
-		if($record = $result->fetchAll(SQLITE3_ASSOC)) 
+		while($arr=$result->fetchArray(SQLITE3_ASSOC))
```
Comment (roda0313) [inline API/API.php]: Did you mean to change this when its not your group? Enrollment changed this maybe an hour ago

Comment (ZacharyMoran) [inline API/API.php]: ^

Comment (ShaydeNofziger) [inline API/API.php]: Looks like the last change I can see around this was 5 hours ago and doesn't actually fix the issue. fetchArray returns 1 row at a time until the query is finished. The call as it was was only returning the first record found from the query. This returns a collection, as the request implies...

Comment (ZacharyMoran) [inline API/API.php]: I havent pushed yet, but its fixed, i thought i had pushed my bad

Comment (ShaydeNofziger) [inline API/API.php]: Alright, we can wait on the PR for yours if you'd like....or the other way around...doesn't matter to me. Going to have to resolve conflicts either way.

File: API/API.php [hunk 2]
```diff
@@ -1766,7 +1801,8 @@ function getStudentGrades()
 	);
 }
 
-//return JSON array
-exit(json_encode($result));
+//return JSON
+header('Content-type:application/json;charset=utf-8');
+echo json_encode($result);
```
Comment (ZacharyMoran) [inline API/API.php]: is this the proper way to do this? the other way followed the examples from class

**Pull request merged**